src/System.Runtime/tests/Performance/Perf.StringBuilder.cs

```diff
@@ -2,61 +2,74 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.Text;
-using Xunit;
+
 using Microsoft.Xunit.Performance;
+using Xunit;
 
 namespace System.Runtime.Tests
 {
-    public class Perf_StringBuilder
+    public static class Perf_StringBuilder
     {
         [Benchmark]
-        public void ctor()
+        public static void Ctor()
         {
-            StringBuilder builder;
+            StringBuilder sb;
```

Comment:
I just follow what the majority of tests in the project follow - basically all the StringBuilder.cs test files use sb so I was being consistent